recipes/bzip2/all/conanfile.py

```diff
@@ -45,27 +41,33 @@ def configure(self):
         del self.settings.compiler.cppstd
 
     def source(self):
-        tools.get(**self.conan_data["sources"][self.version], destination=self._source_subfolder, strip_root=True)
-
-    def _configure_cmake(self):
-        if self._cmake:
-            return self._cmake
-        self._cmake = CMake(self)
-        self._cmake.definitions["BZ2_VERSION_STRING"] = self.version
-        self._cmake.definitions["BZ2_VERSION_MAJOR"] = tools.Version(self.version).major
-        self._cmake.definitions["BZ2_BUILD_EXE"] = self.options.build_executable
-        self._cmake.configure()
-        return self._cmake
+        tools.get(**self.conan_data["sources"][self.version], destination=self.source_folder, strip_root=True)
+        # FIXME: this is failing after the export export to "src" in local folder
+        try:
+            os.rename("CMakeLists.txt", "src/CMakeLists.txt")
```

Comment:
```suggestion
            tools.rename("CMakeLists.txt", "src/CMakeLists.txt")
```
Is it capable of moving files ?

Reply:
Yes, but only when the target directory is already created.